tee-worker/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts

```diff
@@ -26,7 +26,6 @@ export default {
                 __Unused_evm_account_codes: "Null",
                 __Unused_evm_account_storages: "Null",
                 id_graph: "(LitentryIdentity)",
-                id_graph_stats: "(LitentryIdentity)",
```

Comment:
Would suggest: It's better not to simply remove it. But copy the above example `__Unused_xxx` .